release notes/v0.8.0.md

```diff
@@ -1,43 +1,44 @@
 
-As an experimental module, it still means that the APIs are likely to change, causing breaking changes in future releases -- for example, we're going to move more of our APIs to `async`, now that we have `async`/`await` keyword support, but this also means that your Playwright scripts will also start to work in `xk6-browser` with less and less tweaking of your test scripts. Another point to make here is that we're hoping for rough compatibility of Playwright APIs, so we will support some or all of Playwrights APIs, and also extend them too.
+As an experimental module, some of our APIs are likely to change, causing breaking changes in future releases—for example, now that we have `async`/`await` keyword support, we're going to move most of our APIs to `async`. Changing in APIs also means that your Playwright scripts will also start to work in `xk6-browser` with less and less tweaking of your `xk6-browser` test scripts in the upcoming releases—As per our rough compatibility goal with Playwright APIs.
 
-This still means that browser based tests can only be performed within your own test environments, and it won't be available in the [cloud](https://k6.io/cloud/) (yet). Keep your eyes peeled though on future releases, and the blog to hear more on the public launch of browser testing in the cloud.
+Currently, browser based tests can only be performed within your own test environments, and it won't be available in the [cloud](https://k6.io/cloud/) (yet). Keep your eyes peeled though on future releases, and the blog to hear more on the public launch of browser testing in the cloud.
 
 With this release we are continuing to build and upload the artifacts (binaries) for you to download and work with. In the near future we intend to stop that, and instead we will redirect you to the k6 repo, where you will be able to download the latest binary with browser test functionality. We may also stop releasing the release notes here and move them to the k6 release notes. We will create a blog post outlining these details soon.
 
 
 ## Improvements
 
-- Convert promisified (`.then()`) examples to `async`/`await`. [#703](https://github.com/grafana/xk6-browser/pull/703)
+- Pre-merge: Map k6-browser Async API to Goja. [#683](https://github.com/grafana/xk6-browser/issues/683)
```

Comment:
This is an internal refactoring, i don't know how interested an external user would make of this. Maybe it could go in the internal section?

Reply:
Right, but the release notes are not only for external users. The notes are also for the other internal k6 teams, product, management, etc.

Reply:
Looking at a few other release notes ([k6](https://github.com/grafana/k6/releases), [loki](https://github.com/grafana/loki/releases), [tempo](https://github.com/grafana/tempo/releases), as well as our own) they are geared towards informing the end user.